netbox_routing/models/static.py

```diff
@@ -74,3 +76,10 @@ def __str__(self):
 
     def get_absolute_url(self):
         return reverse('plugins:netbox_routing:staticroute', args=[self.pk])
+
+    def clean(self):
+        super().clean()
+
+        # IPv4 and IPv6 cannot be mixed.
+        if self.prefix.version != self.next_hop.version:
+            raise ValidationError(_('The IP version must be the same for the prefix and next hop.'))
```

Comment:
While I haven't actually tried it, I believe it may technically be possible to set a non IPv4/IPv6 next hop with the opposite AF.  Someone might have to confirm that.  I don't think I want to mix this in.

Reply:
That’s indeed possible, at least for certain platforms: https://www.ietf.org/archive/id/draft-chroboczek-intarea-v4-via-v6-00.html#section-5

Should I remove this check or make it optional by adding a configuration option?

Reply:
I would remove that check IMO